modules/core/src/main/java/org/apache/ignite/internal/ClusterMetricsSnapshot.java

```diff
@@ -380,7 +383,7 @@ public ClusterMetricsSnapshot(ClusterGroup p) {
 
             nonHeapCommitted += m.getNonHeapMemoryCommitted();
 
-            nonHeapUsed += m.getNonHeapMemoryUsed();
+            nonHeapMemoryUsage(node);
```

Comment:
In fact when you fix this issue for node metrics (node.metrics().getNonHeapMemoryUsed();) you're no longer need to have this method here at all because node.metrics().getNonHeapMemoryUsed(); will return a valid value.